### PR TITLE
Acknowledge sponsors

### PR DESCRIPTION
**Pull request by miguelaferreira**
... and remove the cryptographic notice section who's reason to exist is the very political environment Cosmic was born in but has since freed it self from .

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -120,6 +120,12 @@ If you've found an issue that you believe is a security vulnerability in a
 released version of Cosmic, please report it to `int-cloud@schubergphilis.com` with details about the vulnerability, how it
 might be exploited, and any additional information that might be useful.
 
+## Sponsors
+
+The Cosmic team would like to recognize and thank the contributions of the follwoing entities to the success of this systems:
+* [Shuberg Philis](https://www.schubergphilis.com): whom is responsible for the birth of the Cosmic project (and all otehr related projects, see MissionCriticalCloud/vagrant-cloudstack for an example) and resulting systems, and is relentlessly supporting all the development costs of Cosmic.
```
Comment (bheuvel) [inline README.md]: follwoing ==> following
"this systemS" ==> "this system" or "these systems"
Shuberg ==> Schuberg
otehr ==> other
Also think whom ==> who

But good one indeed, 👍

Comment (miguelaferreira) [reply README.md]: Thanks @bheuvel. I'll fix the typos

**Comment by miguelaferreira** [comment]
ping @bheuvel

File: README.md [hunk 2]
```diff
@@ -120,6 +120,12 @@ If you've found an issue that you believe is a security vulnerability in a
 released version of Cosmic, please report it to `int-cloud@schubergphilis.com` with details about the vulnerability, how it
 might be exploited, and any additional information that might be useful.
 
+## Sponsors
+
+The Cosmic team would like to recognize and thank the contributions of the follwoing entities to the success of this project and resulting system:
```
Comment (bheuvel) [inline README.md]: follwoing ==> following (deja-vu 😉 ?)

**Comment by bheuvel** [comment]
👍  (will merge as soon as Jenkins finished testing this change 😉 )

**Comment by miguelaferreira** [comment]
fingers crossed :)

**Pull request merged**